packages/framework/ai-collab/package.json

```diff
@@ -0,0 +1,141 @@
+{
+	"name": "@fluid-experimental/ai-collab",
+	"version": "2.4.0",
+	"description": "Experimental package to simplify integrating AI into Fluid-based applications",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/framework/ai-collab"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/public.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/public.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
+		"./alpha": {
+			"import": {
+				"types": "./lib/alpha.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/alpha.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
```

Comment:
Can (I think recommended even) skip /alpha for experimental packages especially if there are no `@internal` APIs which is expected in most experimental cases. Root (public) exports can just use index.d.ts.